Disambiguate error message for working with not fully refined tuple types

tugsbayasgalan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#55745 Disambiguate error message for working with not fully refined tuple types**

Differential Revision: [D27698691](https://our.internmc.facebook.com/intern/diff/D27698691)

tugsbayasgalan committed:
Disambiguate error message for working with not fully refined tuple types

[ghstack-poisoned]

gmagogsfm commented on torch/csrc/jit/frontend/sugared_value.cpp:
```diff
@@ -238,6 +238,9 @@ std::vector<std::shared_ptr<SugaredValue>> SimpleValue::asTuple(
     return fmap(unpack->outputs(), make_simple_value);
+  } else if (value_->type()->kind() == TypeKind::AnyTupleType) {
+    throw ErrorReport(loc)
+        << "Provided tuple is not fully refined to types like Tuple[int, int]";
```
Thanks for the fix! 
```suggestion
        << "Provided tuple is not fully defined/refined including its element types, please provide a value of type like Tuple[int, int]";
```

tugsbayasgalan replied:
done

tugsbayasgalan committed:
Update on "Disambiguate error message for working with not fully refined tuple types"


Differential Revision: [D27698691](https://our.internmc.facebook.com/intern/diff/D27698691)

[ghstack-poisoned]
